services/task/backup.rb

```diff
@@ -0,0 +1,29 @@
+class BackUp
+
+  attr_accessor :path, :credentials, :absolute_path
+  MAX_COUNT_BACKUPS = 3
+
+  def initialize(path = File.join(Dir.home, "backUP"))
+    @path = path
+    @credentials = Rails.application.config.database_configuration[Rails.env]
+    @absolute_path = "#{ path }/#{ Time.now.strftime("%Y%m%d%H%M%S") }.sql"
+  end
+
+  def perform!
+    remove_oldest
+    backup_db
+  end
+
+  def remove_oldest
```

Comment:
I expect this method to clean up old backups and keep only 3 latest dumps, not just the last one